Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSLoginWindowController.mm

```diff
@@ -263,7 +263,7 @@ - (void)loginAttemptEndedWithResult:(ENetError)result
 
 - (IBAction)needAccountButtonHit:(id)sender
 {
-    NSString* urlString = [NSString stringWithSTString:GetServerSignupUrl()];
+    NSString* urlString = ST2NSString(GetServerSignupUrl());
```

Comment:
Another leak - I'm going to stop calling out the leaks individually because there are a few now.